modules/i18n/src/translations/fr-FR/portals/myaccount.ts

```diff
@@ -669,6 +669,32 @@ export const myAccount: MyAccountNS = {
                 }
             }
         },
+        "loginVerifyData": {
```

Comment:
Use google translate to translate the lang namespaces. Check other languages as well.